sys/drv/plic.c

```diff
@@ -13,8 +13,30 @@
  *   https://github.com/riscv/riscv-plic-spec
  */
 
-/* PLIC memory map. */
+/*
+ * NOTE: FTTB, we designate a single HART to handle interrupts from all
+ * peripheral-level devices.
+ *
+ * For the 32-bit platform, the handling HART is the first one:
```

Comment:
This requires some more explanation. Could you refer a reader to the documentation?

Reply:
At the beginning of this file is the following note:
```
/*
 * Details of the operation of PLIC as well as the memory layout
 * can be found in the official specification which is maintained on github:
 *   https://github.com/riscv/riscv-plic-spec
 */
```
Isn't that sufficient?

Reply:
I changed the description to include a link to the original fu540 manual.